packages/cardano-services/src/Program/options/common.ts

```diff
@@ -91,4 +100,19 @@ export const withCommonOptions = (command: Command, defaults: { apiUrl: URL }) =
         .env('SERVICE_DISCOVERY_TIMEOUT')
         .default(SERVICE_DISCOVERY_TIMEOUT_DEFAULT)
         .argParser((interval) => Number.parseInt(interval, 10))
+    )
+    .addOption(
+      new Option('--handle-provider-policy-id <policyId>', CommonOptionDescriptions.PolicyId)
```

Comment:
The new options shouldn't be in 'common' options, but only for the "provider server". "Common" options are the ones applicable to all services: provider server, workers, projector

Also, let's align the naming with [the existing projector option](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/cardano-services/src/cli.ts#L162), and accept an array of policy IDs:
- you can just throw a NotImplementedError if >1 is passed for now
- would be nice to extract a util like `withHandlePolicyIds`

Reply:
3da615fdee6e0139e13a22d9b39ec1fb8ea9499f should fix this (also, I'm aware I rebased, but I wanted the latest from master because the cli test were breaking on stuff I hadn't touched ... not sure the rebase fixed it or not, but after that I could see the breakages from my changes).